packages/manager/apps/pci-databases-analytics/src/pages/services/[serviceId]/backups/fork/_components/ForkSummary.component.tsx

```diff
@@ -139,10 +140,10 @@ const EngineDetails = ({ order }: ForkSummaryProps) => {
             <span>
               {humanizeEngine(order.engine.name as database.EngineEnum)}
             </span>
-            <img
-              className="inline-block w-9 h-6"
-              src={`./assets/engines/${order.engine.name}.png`}
-              alt={order.engine.name}
+            <EngineIcon
+              engine={order.engine.name as database.EngineEnum}
```

Comment:
Why not typing explicitly `order.engine.name` as `database.EngineEnum` instead of a `string` ?